src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -89,13 +101,17 @@
     (let [environment (get-in req [:params "environment"])
           code-id (get-in req [:params "code_id"])
           file-path (get-in req [:params :rest])]
-      (if (or (nil? environment)
-              (nil? code-id)
-              (or (nil? file-path) (= "" file-path)))
+      (cond
+        (some empty? [environment code-id file-path])
         {:status 400
          :body "Error: A /static_file_content request requires an environment, a code-id, and a file-path"}
+        (not (valid-static-file-path? file-path))
+        {:status 403
+         :body (str "Request Denied: A /static_file_content request must be a file within "
```

Comment:
As above, I think the intention of this endpoint is to only let a user make a request against a specific environment, so we might want to change this to something like `Request Denied: A /static_file_content request must be made for a file within the files directory of a module`